src/expire.c

```diff
@@ -275,10 +276,15 @@ static long long activeExpireCycleJob(enum activeExpiryType jobType, int cycleTy
             case KEYS:
                 kvs = db->expires;
                 scan_cb = expireScanCallback;
+                time_check_rate = 0xf; /* For regular keys we can check the time condition every 16 loop iterations */
                 break;
             case FIELDS:
                 kvs = db->keys_with_volatile_items;
                 scan_cb = fieldExpireScanCallback;
+                time_check_rate = 0x0; /* For field-level keys we check the time condition every loop iteration.
+                                        * This is required since we might perform much more operation per single key with many fields.
+                                        * Limiting the number of fields we scan in each field makes the overall process less efficient.
+                                        * So we just perform more clock checks after each iteration. */
```

Comment:
```suggestion
                /* For field-level keys we check the time condition every loop iteration.
                 * This is required since we might perform much more operation per single key with many fields.
                 * Limiting the number of fields we scan in each field makes the overall process less efficient.
                 * So we just perform more clock checks after each iteration. */
                time_check_rate = 0x0;
```
Reduces indentation.